### PR TITLE
feat: add voiceover to slides (with some help from @Phreakester)

### LINKED ISSUE #83: 🚀 Feature: Add in voice-over for all animations and carousel slides
* [x] :wave: I may be able to implement this feature request

---

This is a :rocket: Feature Request

### PR DESCRIPTION
**Pull request by reginawang99**
closes #83

### CODE REVIEW

File: src/components/shared/Carousel.tsx [hunk 1]
```diff
@@ -38,27 +41,32 @@ function Carousel(props: CarouselProps): JSX.Element {
...
+    const muted = storage.getItem('isMuted');
+    if (!muted) return;
+    setIsMuted(true);
+  }, []);
+
+  useEffect(() => {
+    if (!storage.getItem('isAutoAdvance')) return;
+    setIsAutoAdvance(true);
+    if (child.animationTime) { autoAdvance(child.animationTime); }
+  }, []);
```
Comment (BryanPan342) [inline src/components/shared/Carousel.tsx:67]: where have these all separated?

**Review by BryanPan342 — DISMISSED** [review]
Uno requesto

File: src/components/shared/Carousel.tsx (hunk 1 above)
Comment (reginawang99) [reply src/components/shared/Carousel.tsx:67]: oui ill put back together!

**Comment by reginawang99** [comment]
- mute doesn't work if you go to a new carousel (for voice over)
- carousel dismount doesn't stop audio
- a bit funked slide
image

**Comment by reginawang99** [comment]
fixed 1.

2 & 3 are still messed up 
@BryanPan342 thoughts on just making fix tickets?

**Comment by reginawang99** [comment]
fixed 1 & 2, will make a ticket for #3

**Review by tr89on — APPROVED** [review]
approved!

**Pull request merged**